### PR TITLE
New package: flrig-1.4.7

### PR DESCRIPTION
**Pull request by Goorzhel**
- I tested the changes in this PR: **briefly**
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

### CODE REVIEW

File: srcpkgs/flrig/template [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+pkgname=flrig
+version=1.4.7
+revision=1
+build_style=gnu-configure
+makedepends="fltk-devel"
+depends="fltk"
+short_desc="Rig control program, compatible with fldigi"
+maintainer="Antonio Gurgel <antonio@goorzhel.com>"
+license="GPL-3.0-or-later"
+homepage="http://www.w1hkj.com/"
```
Comment (classabbyamp) [inline srcpkgs/flrig/template:11]:
```suggestion
homepage="http://www.w1hkj.com/"
changelog="https://sourceforge.net/p/fldigi/flrig/ci/master/tree/ChangeLog?format=raw"
```

File: srcpkgs/flrig/template [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Template file for 'flrig'
+pkgname=flrig
+version=1.4.7
+revision=1
+build_style=gnu-configure
+makedepends="fltk-devel"
+depends="fltk"
+short_desc="Rig control program, compatible with fldigi"
+maintainer="Antonio Gurgel <antonio@goorzhel.com>"
+license="GPL-3.0-or-later"
```
Comment (classabbyamp) [inline srcpkgs/flrig/template:10]: license is unclear, there's a COPYING file that is GPL 2, but src/main.cxx says GPL 3+. Other distros use a [variety of values](https://repology.org/project/flrig/information). Could you investigate this?

Comment (Goorzhel) [reply srcpkgs/flrig/template:10]: The COPYING is v3:
```
/tmp/fldigi-4.1.23 ❯ head -2 COPYING
                    GNU GENERAL PUBLIC LICENSE
                       Version 3, 29 June 2007
```
But it ships mbedtls in `src/mbedtls` under GPL v2. Thoughts?

Comment (classabbyamp) [reply srcpkgs/flrig/template:10]: weird, that's not what [this shows](https://sourceforge.net/p/fldigi/flrig/ci/master/tree/COPYING)

Comment (Goorzhel) [reply srcpkgs/flrig/template:10]: I've emailed the author.

Comment (Goorzhel) [reply srcpkgs/flrig/template:10]: He wrote:
> GPL-v3 or later is the intended license for all of the NBEMS suite applications including
>  * fldigi / flarq
>  * flrig
>  * flcluster
>  * flmsg
>  * flamp
>  * fllog
>  * flwrap
>  * flwkey
>  * flnet
>  * flaa
>  * flxmlrpc
>  * flnano
>  * linsim
>  * comptty
>  * comptext
>  * pistats

Comment (classabbyamp) [reply srcpkgs/flrig/template:10]: ok, sounds good

**Comment by classabbyamp** [comment]
the other members of the fl suite might be nice to have too

**Pull request merged**